src/lib/types.ts

```diff
@@ -523,9 +523,9 @@ export type GenerateLinkType =
 
 // MFA related types
 export type MFAEnrollParams = {
-  friendlyName?: string
   factorType: 'TOTP'
```

Comment:
```suggestion
  factorType: 'totp'
```

Late update on my end but Server side now takes `totp` as a lowercased values so we can probably do the same in the lib